src/main/resources/edu/cornell/kfs/core/documentstore/InstitutionPreferences.json

```diff
@@ -134,8 +137,16 @@
           }
         }
       }, {
-        "businessObjectClass": "org.kuali.kfs.gl.businessobject.CurrentAccountBalance",
-        "linkType": "kfs"
+        "label": "Current Fund Balance",
+        "link": "kr/lookup.do?methodToCall=start&businessObjectClassName=edu.cornell.kfs.gl.businessobject.CurrentAccountBalance&docFormKey=88888888&hideReturnLink=true",
+        "linkType": "kfs",
+        "permission": {
+          "templateNamespace": "KR-NS",
+          "templateName": "Use Screen",
```

Comment:
Given that this is also a lookup, should this be a "Look Up Records" permission as well, or should it be a "Use Screen" one like it is currently? Sorry for not pointing it out the first time, didn't think it was a problem at the time.